docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx

```diff
@@ -0,0 +1,172 @@
+| ods-admin-api-db | PostgreSQL 13.12 | Pre-configured PostgreSQL server with two database: EdFi\_Admin with Admin Api specific tables and EdFi\_Security. |
+| PgBouncer | PgBouncer latest (1.20.0) | Lightweight PostgreSQL connection pooler that [improves application scalability](https://www.percona.com/blog/2018/06/27/scaling-postgresql-with-pgbouncer-you-may-need-a-connection-pooler-sooner-than-you-expect/) through connection pooling. |
+
+### Additional Images
+
+These Ed-Fi Alliance maintained images are not included in the configurations
+described below, and therefore must be deployed separately.
+
+| Image Name | Application | Purpose |
+| --- | --- | --- |
+| data-import | Ed-Fi Data Import | Hosts the Ed-Fi Data Import web application, configured for use   with PostgreSQL as a backing database. <br/>|
+| analytics-middle-tier | Ed-Fi Analytics Middle Tier | Distribution of the Analytics Middle Tier command line installation tool. |
+
+:::tip
+See [Docker Deployment for Data
+Import](https://edfi.atlassian.net/wiki/spaces/EDFITOOLS/pages/24119478/Docker+Deployment+for+Data+Import)
+for usage instructions
+:::
+
+## Configuration Options
+
+The following configurations are supported out of the box. In the diagrams
+below, the dashed line represents the internal Docker network, and all
+rectangles represent separate containers. Cylinders represent data that should
+be stored in volumes that are mapped outside of the Docker network for
+permanency.
+
+### Sandbox Configuration
+
+A Sandbox environment is generally used to support API client developers in
+developing client applications. It is not intended to be a staging environment
+for the platform host. Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-Fi Sandbox Admin
+* SwaggerUI
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![Sandbox Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-PostgreSQL.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-PostgreSQL.png)
+  </TabItem>
+  <TabItem value="SQL Server" label="SQL Server">
+    ![Sandbox Configuration - SQL Server](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-Mssql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-Mssql.png)
+  </TabItem>
+</Tabs>
+
+
+### SingleTenant Configuration
+
+This configuration shows a basic example of a single tenant configuration where
+a single ODS database serves all data going through the API. While this example
+has one ODS for demonstration purpose, single tenant deployments can support
+separate ODS per school years and districts. See [API Client and ODS Instance
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559631/API+Client+and+ODS+Instance+Configuration)
```

Comment:
update this link to /reference/ods-api/platform-dev-guide/configuration/api-client-and-ods-instance-configuration